networkx/algorithms/isomorphism/isomorph.py

```diff
@@ -73,12 +73,12 @@ def fast_could_be_isomorphic(G1,G2):
     if G1.order() != G2.order(): return False
 
     # Check local properties
-    d1=G1.degree()
+    d1=dict(G1.degree())
     t1=nx.triangles(G1)
     props1=[ [d1[v], t1[v]] for v in d1 ]
```

Comment:
Should probably take out the ```dict``` you put in line 76 and revamp line 78 as

    props1=[ [d, t1[v]] for v,d in d1 ]

Same sort of thing for d2 and props2 below.